src/SFA.DAS.FindEmployerSchemes.IntegrationTests/Web/Models/SchemeDetailsModelTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using AutoFixture;
+using AutoFixture.Kernel;
+using Microsoft.AspNetCore.Html;
+using SFA.DAS.FindEmploymentSchemes.Web.Models;
+using Xunit;
+
+namespace SFA.DAS.FindEmploymentSchemes.IntegrationTests.Web.Models
+{
+    public class SchemeDetailsModelTests
```

Comment:
Think this belongs with the unit tests too.

Reply:
Removed